boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -214,6 +214,9 @@ if(USE_CLANG_CL)
     # We need to reduce the binary size
     target_compile_options(freeldr_common PRIVATE "/Os")
 endif()
+if(CMAKE_C_COMPILER_ID STREQUAL "GNU" AND (CMAKE_C_COMPILER_VERSION VERSION_GREATER_EQUAL 9))
+    target_compile_options(freeldr_common PRIVATE -Wno-format)
```

Comment:
Is it whining about `%wZ`?
In that case, it would be worth investigating if we can teach it about that.

Reply:
If it was about `%wZ`, then why would this work elsewhere in our code?